@here/harp-mapview/lib/FrustumIntersection.ts

```diff
@@ -129,49 +145,76 @@ export class FrustumIntersection {
     ): FrustumIntersection.Result {
         this.m_tileKeyEntries.clear();
         let calculationFinal = true;
+        const targetTileArea = Math.pow(256 / this.mapView.focalLength, 2);
```

Comment:
Currently, the overall zoom level is chosen depending on the distance from the camera to the ground so that the screen space area of a tile is roughly 256x256. See: `MapViewUtils.calculateZoomLevelFromDistance()`. This only works for Mercator projection and top-down views(camera tilt = 0). For spherical projection and/or tilted views the screen space size varies per tile. The `targetTileArea` is calculated in a similar fashion so that each tile with mixed LOD enabled is also roughly 256x256 on screen(i.e. a higher zoom level is used for tiles further away or tiles that a close to the poles).